### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by bz154** [hyzhang98/EGAE#1]
I would like to ask whether these indicators are obtained from the test set. Does graph clustering need to divide test set and training set

**Comment by hyzhang98** [comment]
> I would like to ask whether these indicators are obtained from the test set. Does graph clustering need to divide test set and training set

No. The setting is the same as the classical clustering. There is no training/validation/test split.

**Comment by luckykidsGL** [comment]
Can you upload wiki.mat data? I am improving on your article and hope to upload the data set to this asset. Thank you very much！

**Comment by hyzhang98** [comment]
Hi, I downloaded the dataset from https://github.com/GRAND-Lab/MGAE/blob/master/data/mgae_wiki.mat. Hope it helps you.

**Comment by arthuraal** [comment]
How were those metric values ​​in the paper obtained? In the repository, the parameters were set to values ​​different from those presented in the paper. Thank you very much.
